### PR TITLE
totem: add new homepage

### PR DESCRIPTION
**Pull request by androidnisse**
**Summary**
- Add new homepage

**Test Plan**
- Checked homepage was changed.

**Checklist**

- [X] Package was built and tested against unstable

### CODE REVIEW

File: packages/t/totem/package.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 name       : totem
 version    : '43.0'
-release    : 32
+release    : 33
 source     :
     - https://download.gnome.org/sources/totem/43/totem-43.0.tar.xz : b36d3655928b5896462a4d39f83b6ad66d37dbc9c99dceb02ea8a2e37394a41e
-homepage   : https://wiki.gnome.org/Apps/Videos
+homepage   : https://apps.gnome.org/en-GB/Totem/
```
Comment (malfisya) [inline packages/t/totem/package.yml:6]: I think leaving the `/en-GB/` would be better. It still works too

**Review by malfisya — CHANGES_REQUESTED** [review]
Just one small nit.

**Review by malfisya — CHANGES_REQUESTED** [review]
Didn't see it before but it needs a rebase too.

**Comment by androidnisse** [comment]
Something has gone wrong here so better start over.

**Pull request closed without merging**